commit 635437c7bfe6053dcfe112705b809bb78aba1a73
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Mon Feb 15 13:42:36 2010 +0200

    Unbreak find-requires (#443015, #564527)
    - regression originating from commit 9ed9b4e3459e3125befd324f579f751a239c26ca
      which was supposed to fix something for on ARM but broke pretty much
      all else
    - this should've been in 9.1.0 but somehow gone missing, ugh...
    - fix originally from Bill Nottingham

diff --git a/find-requires b/find-requires
index fa5ab95..eb3ee54 100755
--- a/find-requires
+++ b/find-requires
@@ -106,6 +106,7 @@ for f in $liblist $exelist ; do
 		print $2 ;
 	    }
 	}
+	(START==2) && /^[A-Za-z]/ { START=3; }
 	/^Version References:$/ { START=2; }
 	(START==2) && /required from/ {
 	    sub(/:/, "", $3);
@@ -114,7 +115,6 @@ for f in $liblist $exelist ; do
 	(START==2) && (LIBNAME!="") && ($4!="") {
 	    print LIBNAME "(" $4 ")'$lib64'";
 	}
-	/^[A-Za-z]/ { START=3; }
     '
 done | sort -u
 
